### DISCUSSION TITLE
Is it normal to take a while to approve templates?

### DISCUSSION THREAD

**Discussion by leonardoefel** [netbox-community/devicetype-library#1738] [General]
I did this pull two weeks ago and didn't get approval, and I saw some that were done later were already approved, I don't know if I did something wrong. thanks!

https://github.com/netbox-community/devicetype-library/pull/1712

**Comment by danner26** [comment]
Hello, this is a community maintained repo. Unfortunately my day job has had me busy. I have finally caught up on the numerous PRs and it looks like yours still has some issues. Feel free to chat with me about it there if you need more assistance
